### PR TITLE
Transitioning MRB page to use custom services.

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: nar_common/static/nar_ui/time_series/TimeSeries.js [hunk 1]
```diff
@@ -33,6 +33,10 @@ nar.timeSeries.TimeSeries = function(config){
 	                ];
 	                return dateAndValues;
 	            });
```
Comment (cschroed-usgs) [inline nar_common/static/nar_ui/time_series/TimeSeries.js]: Note that the data retrieval services are not returning responses sorted by time.

Comment (cschroed-usgs) [inline nar_common/static/nar_ui/time_series/TimeSeries.js]: @jiwalker-usgs: Currently the DB queries for retrieval are sorting based on some stuff besides time. Example:
https://github.com/USGS-CIDA/nar-services/blob/master/nar-webservices/src/main/resources/gov/usgs/cida/nar/mybatis/mappers/AloadsMapper.xml#L53

Is there any danger in switching those queries to sort by time alone, or by time first? If we did that then clients wouldn't have to do the sorting. I can't think of any danger, but maybe there's something in the download that relies on that convention.

Comment (jiwalker-usgs) [inline nar_common/static/nar_ui/time_series/TimeSeries.js]: I wasn't sure about this when I wrote the queries, but I followed the approximate order of the downloads.  Basically if you request multiple constituents (or other) then you get it sorted first by that, then by time.  This is shared with the download currently so we shouldn't remove them or the downloads will come back non-deterministically.  I think modtype is the main thing that is causing issues, so possibly that could be removed.

Comment (cschroed-usgs) [inline nar_common/static/nar_ui/time_series/TimeSeries.js]: Good suggestion! Thanks!

**Pull request merged**